Makefile

```diff
@@ -86,3 +86,13 @@ clean:
 	find . -type d -name "__pycache__" -delete
 	rm -f .coverage tags
 	rm -rf examples/convert/{main.fmf,test.md,Manual}
+
+requre_install:
+	pip install git+https://github.com/packit/requre.git
+
+# Regenerate test data for integration tests
+# remove selected/all response files in tests/integration/test_data directory
+regenerate_integration_tests: requre_install
```

Comment:
What about grouping this with the other test targets in the `Makefile` and use a shorter name? Perhaps `integration` or `requre` would be enough?

Reply:
okay